reverse_engineering/helpers/oracleHelper.js

```diff
@@ -834,8 +834,8 @@ const getDbSynonyms = async logger => {
 		logger.log('info', synonyms, 'Getting synonyms');
 		const synonymsDDL = await getSynonymsDDL();
 		const synonymsWithEditionable = synonyms.map(synonym => {
-			const synonymDDL = synonymsDDL.find(ddl => ddl.includes(`"${synonym.synonymName}" FOR`));
-			const isEditionable = !synonymDDL.includes('NONEDITIONABLE');
+			const synonymDDL = synonymsDDL.find(({ name }) => name === synonym.synonymName);
+			const isEditionable = !synonymDDL.ddl.includes('NONEDITIONABLE');
```

Comment:
could you please also make this check case insensitive, because different implementation of DB instance not guarantee that NONEDITIONABLE label will be always in uppercase?